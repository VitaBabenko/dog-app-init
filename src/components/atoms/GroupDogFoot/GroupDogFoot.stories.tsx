import type { Meta, StoryObj } from '@storybook/react';
import { GroupDogFoot } from '.';

const meta = {
  title: 'Atoms/GroupDogFoot',
  component: GroupDogFoot,
  argTypes: {
    state: {
      options: ['default'],
      control: 'radio'
    }
  }
} satisfies Meta<typeof GroupDogFoot>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Primary: Story = {
  args: {
    state: 'default'
  }
};
