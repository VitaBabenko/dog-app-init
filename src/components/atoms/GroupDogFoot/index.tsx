import { SvgIcon } from '@mui/material';

import DogFootGroup from '../../../assets/icons/GroupDogFoot.svg?react';

export const GroupDogFoot = () => (
  <SvgIcon
    component={DogFootGroup}
    inheritViewBox
    sx={{ width: 1408, height: 74 }}
  />
);
