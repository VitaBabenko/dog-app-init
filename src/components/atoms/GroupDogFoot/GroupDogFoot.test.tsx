import { composeStories } from '@storybook/react';
import * as stories from './GroupDogFoot.stories';
import { testSnapshot } from '../../../test/utils/snapshot.utils';

describe('GroupDogFoot', () => {
  const { Primary } = composeStories(stories);
  testSnapshot(<Primary />);
});
