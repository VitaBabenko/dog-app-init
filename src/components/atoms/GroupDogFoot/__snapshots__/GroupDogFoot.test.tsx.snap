// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupDogFoot should match snapshot Component 1`] = `
<div>
  <test-file-stub
    aria-hidden="true"
    classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-c6m6ws-MuiSvgIcon-root"
    focusable="false"
  />
</div>
`;
